packages/terra-worklist-data-grid/src/subcomponents/ColumnHeaderCell.jsx

```diff
@@ -183,19 +187,28 @@ const ColumnHeaderCell = (props) => {
   // Retrieve current theme from context
   const theme = useContext(ThemeContext);
 
+  const pinnedColumnsDivider = columnIndex === columnContext.pinnedColumnsLength - 1
+    ? (
+      <div
+        className={cx('pinned-columns-divider')}
+        style={{ height: tableHeight, left: width - 1 }} // eslint-disable-line react/forbid-dom-props
```

Comment:
You should be able to set the left style with CSS, right?  You don't need that here.

Reply:
Can you elaborate? This is dynamically calculated based on the width of the last pinned column.

Reply:
ok.  I misunderstood the code.